dev-docs/howtoguides/testing.md

```diff
@@ -67,8 +67,8 @@ tox -e behave
 or, if you just want to run a specific file, or a test within a file:
 
 ```shell
-tox -e behave features/unattached_commands.feature
-tox -e behave features/unattached_commands.feature:28
+tox -e behave -- features/unattached_commands.feature
```

Comment:
very strictly speaking, we could 
```suggestion
tox -e behave -- -i features/unattached_commands.feature
```
wdyt

Reply:
`-i` is for a regex of files
position argument is an exact file

So I think as-is is good